jdaviz/configs/specviz/plugins/line_analysis/line_analysis.py

```diff
@@ -288,6 +288,10 @@ def _uncertainty(result):
             # don't need these if statements
             if function == "Line Flux":
                 flux_unit = spec_subtracted.flux.unit
+                add_flux = False
+                if str(flux_unit) == '':
```

Comment:
is there any benefit to checking this instead of against `u.dimensionless_unscaled`?

Reply:
No benefit, I agree that it really should be` u.dimensionless_unscaled`